src/arrays.js

```diff
@@ -1,59 +1,77 @@
 export const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length) {
+    return array[index % array.length]
+  } 
+  return array[index]
 };
 
 export const arrayToCSVString = (array) => {
-  // your code here
+  return array.toString()
 };
 
 export const csvStringToArray = (string) => {
-  // your code here
+  return string.split(',')
 };
 
-export const addToArray = (array, element) => {
-  // your code here
+export const addToArray = (element, array) => {
+  array.push(element)
 };
 
-export const addToArray2 = (array, element) => {
-  // your code here
+export const addToArray2 = (element, array) => {
+  const array2 = array.concat(element) 
+  return array2
 };
 
 export const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1)
 };
 
 export const numbersToStrings = (numbers) => {
-  // your code here
+  return numbers.map(number => number.toString())
 };
 
 export const uppercaseWordsInArray = (strings) => {
-  // your code here
+  return strings.map(function(string) {
+    return string.toUpperCase() 
+  })
 };
 
 export const reverseWordsInArray = (strings) => {
-  // your code here
+  return strings.map(string => string.split('').reverse().join(''))
 };
 
 export const onlyEven = (numbers) => {
-  // your code here
+  return numbers.filter(function(num) {
+    return num % 2 === 0
+  })  
 };
 
 export const removeNthElement2 = (index, array) => {
-  // your code here
+  const array2 = array.slice(0, index)
+  const array3 = array.slice((index + 1))
+  const array4 = array2.concat(array3)
+  return array4  
```

Comment:
It's very good that you've left the original array as is here (i.e. you haven't mutated it). `array2`, `array3` and `array4` don't read very well though - perhaps some more explanatory names?

Also you don't need to assign to `array4`. You can just `return array2.concat(array3)` :)